breathecode/media/migrations/0007_alter_mediaresolution_media.py

```diff
@@ -0,0 +1,20 @@
+# Generated by Django 3.2.3 on 2021-06-05 18:50
```

Comment:
Try to add just one migration per app, we can check the new migrations that is not in production, then remove it and run `pipenv run makemigrations` for send it in just one migration the same number of changes